docs/BreakingChanges/149 SSL 2.0 and 3.0 now obsolete in SslProtocols and SecurityProtocolType enums.md

```diff
@@ -0,0 +1,37 @@
+## 149: SSL 2.0 and 3.0 now obsolete in SslProtocols and SecurityProtocolType enums
+
+### Scope
+Major
+
+### Version Introduced
+4.6.2
+
+### Source Analyzer Status
+Planned
+
+### Change Description
+In .NET Framework 4.6.2, The following enums are marked as obsolete
+because SSL 2.0 and SSL 3.0 have been shown to be insecure protocols.
+
+- `System.Security.Authentication.SslProtocols.Ssl2`
+- `System.Security.Authentication.SslProtocols.Ssl3`
+- `System.Security.Authentication.SslProtocols.Default` which is
+  defined as `System.Security.Authentication.SslProtocols.Default | System.Security.Authentication.SslProtocols.Tls`
+- `System.Net.SecurityProtocolType.Ssl3`
+
+Developers will see compile warnings for these enums (or compile
+errors if Treat Warnings as Errors option is turned on) when they
```

Comment:
nit: Bold or put quotation marks around Treat Warnings as Errors to highlight that it's an option.